docs/upgrade/cluster-restart.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: default
+title: Cluster Restart Upgrade
+parent: Upgrade
+nav_order: 2
+---
+
+# Cluster restart upgrade
+
+The steps on this page are most applicable if you installed Open Distro for Elasticsearch using the RPM or Debian packages. If you used a Docker image, see [Docker upgrade](../docker/).
+
+1. Disable shard allocation to prevent Elasticsearch from replicating shards as you shut down each node:
+
+   ```json
+   PUT _cluster/settings
+   {
+     "persistent": {
+       "cluster.routing.allocation.enable": "primaries"
+     }
+   }
+   ```
+
+1. Stop Elasticsearch on each node:
+
+   ```bash
+   sudo systemctl stop elasticsearch.service
+   ```
+
+1. Upgrade packages on each node using `yum` or `apt`:
+
+   ```bash
```

Comment:
Hey @allenyin96, should we be recommending that people install and upgrade to specific versions rather than the vanilla "latest" command?

Reply:
Yes, let's make clear the specific version being installed and the breaking changes that might come with it.